sass/style.css

```diff
@@ -45,6 +57,33 @@ a {
   background: white;
 }
 
+.search-filter-container .bg-dark-blue {
+  padding: 2rem !important;
+}
+
+.search-filter-container h4 {
+  font-weight: 700;
+  font-size: 22px;
```

Comment:
Should use rem units throughout.

Reply:
Lol I started fixing this, then noticed theres px sizes everywhere from before, so imma just leave it until later when I can transition this file over to using SASS vars